docker/web/dev/Dockerfile

```diff
@@ -12,3 +12,10 @@ RUN apt-get update && \
 
 COPY xdebug.ini /etc/php/${PHP_V}/apache2/conf.d/20-xdebug.ini
 COPY xdebug.ini /etc/php/${PHP_V}/cli/conf.d/20-xdebug.ini
+
+CMD usermod -u $WEB_UID www-data && \
+    su www-data bash -c "cd /var/www/script/migration && composer install --no-dev && ./vendor/bin/phinx migrate" && \
```

Comment:
I think the script should check if migration is actually needed. Maybe first drop the current table?

Reply:
Phinx execute migrations only new migrations.

Reply:
Actually the problem is if the previous table exist. Than you get `Base table or view already exists`. You can test the exit status from `/vendor/bin/phinx migrate`, eventually drop the table and execute migrations.

Reply:
When a new image is built, there is no error. Guess it's ownership issue.